protocol/x/clob/mev_telemetry/telemetry.go

```diff
@@ -40,40 +40,44 @@ func SendDatapoints(ctx sdk.Context, address string, mevMetrics types.MevMetrics
 		}
 	}()
 
-	data, err := json.Marshal(mevMetrics)
+	for _, address := range addresses {
+		sendDatapointsToTelemetryService(ctx, address, mevMetrics)
```

Comment:
the initial reasoning (the first time I submitted this PR, but didn't follow up on getting it merged) was that the panic catcher would not catch panics in `sendDatapointsToTelemetryService` if those requests were in a goroutine. It could be fixed by adding a panic catcher inside the function, but that seemed unnecessary.